Backend/models/Waitlist.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+​
+const WaitlistSchema = mongoose.Schema({
```

Comment:
Is this schema still needed, not that Timeslots includes waitlisted and reserved students?